code/src/pages/CastDetails.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react"
+import { useParams, useHistory } from "react-router-dom"
+
+import { CAST_URL } from '../utils/urls'
+import Loader from "../components/Loader"
+import CastProfile from "../components/CastProfile"
+import HomePath from "../components/HomePath"
+
+const CastDetails = () => {
+  const { id } = useParams()
+  const history = useHistory()
+  const [cast, setCast] = useState()
+
+  useEffect(() => {
+    fetch(CAST_URL(id))
+      .then(res => res.json())
+      .then(data => {
+        if (data.status_message) {
+          history.push("/notfound")
+        } else {
+          setCast(data.cast)
+        }
+      })
+  }, [id, history])
+
+  if (cast === undefined) {
+    return (<Loader />)
+  }
+
+  return (
+    <section className="cast-details-container">
+      <HomePath />
```

Comment:
I think it would be nicer to be sent back to details here, not to the home page 😸